src/index.ts

```diff
@@ -222,7 +222,7 @@ export class Paginator {
       }
 
       // maxResults is the user specified a limit.
-      if (callback && (maxResults !== -1 || query.autoPaginate === false)) {
+      if ((maxResults !== -1 || query.autoPaginate === false)) {
```

Comment:
Does this comment   " // maxResults is the user specified a limit." still hold true? Also, it needs to be rephrased. 
"maxResults is the user-specified limit. "

Reply:
comment still holds true, removed 'a', thanks